src/SprykerEco/Yves/Payone/Controller/KlarnaController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Payone\Controller;
+
+use Generated\Shared\Transfer\PayoneKlarnaStartSessionRequestTransfer;
+use Spryker\Yves\Kernel\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @method \SprykerEco\Client\Payone\PayoneClientInterface getClient()
+ * @method \SprykerEco\Yves\Payone\PayoneFactory getFactory()
+ */
+class KlarnaController extends AbstractController
+{
+    protected const PAY_METHOD_REQUEST_PARAMETER = 'pay_method';
```

Comment:
* `REQUEST_PARAMETER_PAY_METHOD`, the first is the description of the field, then its name.
* Change other constants in that way.

Reply:
++